lib/spoom/sorbet/sigils.rb

```diff
@@ -54,32 +56,56 @@ def update_sigil(content, new_strictness)
           content.sub(SIGIL_REGEXP, sigil_string(new_strictness))
         end
 
+        sig { params(content: String, new_strictness: String).returns(String) }
```

Comment:
Can you document the new methods you added? We try to document all methods in this gem.